server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/UserRoutesTest.java

```diff
@@ -362,7 +362,7 @@ default void deleteShouldStillBeValidWithExtraBody() {
                     Arguments.of(")"),
                     Arguments.of(","),
                     Arguments.of(":"),
-                    Arguments.of(";"),
+//                    Arguments.of(";"),
```

Comment:
I agree to temporarily @disable that test (but we need to extract it in a separate @disabled test rather than commenting it out).

Questions: 

 - is it a REST assured issue? Or a spark.java issue?

Can you run a manual curl test on a memory james and see is ; is allowed ? (you might have to URL-encode it)

Reply:
Seems to be a spark.java issue, when debugging you can see it can't extract the full name from the request, instead of having a `Username("user;name", Optional.of("domain")`, we only end up with `Username("user", Optional.empty)`

everything from the `;` is dropped literally

Will separate it into a @disabled test and add a task about it? (if downgrading to httpcore 5.9 actually solves our issue)

Reply:
https://github.com/linagora/james-project/issues/3767